scripts/artifacts-building/apt/build-apt-artifacts.sh

```diff
@@ -87,14 +87,15 @@ set -x
 # Ensure that the repo server public key is a known host
 grep "${REPO_HOST}" ~/.ssh/known_hosts || echo "${REPO_HOST} $(cat $REPO_HOST_PUBKEY)" >> ~/.ssh/known_hosts
 
-#Append host to [mirrors] group
+# Create an inventory to use
 echo '[mirrors]' > /opt/inventory
 echo "repo ansible_host=${REPO_HOST} ansible_user=${REPO_USER} ansible_ssh_private_key_file='${REPO_KEYFILE}' " >> /opt/inventory
+echo "localhost ansible_python_interpreter: '/usr/bin/python2'" >> /opt/inventory
```

Comment:
I think for static inventories, it's probably better to use the format ansible_python_interpreter='/usr/bin/python2'